.ci/travis/ubuntu/Dockerfile

```diff
@@ -28,7 +30,7 @@ RUN pip install tox
 COPY . /root/ngraph
 RUN mkdir /root/ngraph/build
 WORKDIR /root/ngraph/build
-RUN cmake ../ -DNGRAPH_USE_PREBUILT_LLVM=TRUE -DCMAKE_INSTALL_PREFIX="~/ngraph_dist"
+RUN cmake .. -DNGRAPH_USE_PREBUILT_LLVM=TRUE -DNGRAPH_ONNX_IMPORT_ENABLE=TRUE
```

Comment:
We'll need to keep the `-DCMAKE_INSTALL_PREFIX="~/ngraph_dist"`

Reply:
Have got it back.